src/utils/IO/branchdata_checks.jl

```diff
@@ -59,105 +59,114 @@ function sanitize_angle_limits!(line)
     return
 end
 
-function linerate_calculation(l::Union{Line, MonitoredLine})
-    theta_max = max(abs(l.angle_limits.min), abs(l.angle_limits.max))
-    g = l.r / (l.r^2 + l.x^2)
-    b = -l.x / (l.r^2 + l.x^2)
-    y_mag = sqrt(g^2 + b^2)
-    fr_vmax = isnothing(l.arc.from.voltage_limits) ? nothing : l.arc.from.voltage_limits.max
-    to_vmax = isnothing(l.arc.to.voltage_limits) ? nothing : l.arc.to.voltage_limits.max
-
-    if isa(fr_vmax, Nothing) || isa(to_vmax, Nothing)
-        fr_vmax = 1.0
-        to_vmax = 0.9
-        diff_angle = abs(l.arc.from.angle - l.arc.to.angle)
-        new_rate = y_mag * fr_vmax * to_vmax * cos(theta_max)
-    else
-        m_vmax = max(fr_vmax, to_vmax)
-        c_max = sqrt(fr_vmax^2 + to_vmax^2 - 2 * fr_vmax * to_vmax * cos(theta_max))
-        new_rate = y_mag * m_vmax * c_max
-    end
+# https://scholarspace.manoa.hawaii.edu/bitstream/10125/50237/paper0350.pdf
+# https://www.mdpi.com/1996-1073/10/8/1233/htm
+const MVA_LIMITS_LINES = Dict(
+    69.0 => (min = 12.0, max = 115.0),
+    115.0 => (min = 92.0, max = 255.0),
+    138.0 => (min = 141.0, max = 344.0),
+    161.0 => (min = 176.0, max = 410.0),
+    230.0 => (min = 327.0, max = 797.0),
+    345.0 => (min = 897.0, max = 1494.0),
+    500.0 => (min = 1732.0, max = 3464.0))
+
+# https://scholarspace.manoa.hawaii.edu/bitstream/10125/50237/paper0350.pdf
+# https://www.mdpi.com/1996-1073/10/8/1233/htm
+const MVA_LIMITS_TRANSFORMERS = Dict(
+    69.0 => (min = 7.0, max = 115.0),
+    115.0 => (min = 17.0, max = 140.0),
+    138.0 => (min = 15.0, max = 239.0),
+    161.0 => (min = 30.0, max = 276.0),
+    230.0 => (min = 50.0, max = 470.0),
+    345.0 => (min = 160.0, max = 702.0),
+    500.0 => (min = 150.0, max = 1383.0),
+    765.0 => (min = 2200.0, max = 6900.0), # This value is 3x the SIL value from https://neos-guide.org/wp-content/uploads/2022/04/line_flow_approximation.pdf
+)
 
-    return new_rate
-end
+function check_rating_values(line::Union{Line, MonitoredLine}, basemva::Float64)
```

Comment:
Are we sure that at this point the `rating_value` can not be negative?

Reply:
I also think that if the function is called `check` we need to return `true` or `false`. It looks like it is always returning `true`

Reply:
How can a rating be negative? What's the interpretation of a negative power value for rating?